### ISSUE TITLE
Replace references to `$BITBUCKET_SHARED_HOME` environment variable with `$BITBUCKET_HOME`

### ISSUE THREAD

**Issue by timrogers** [github/gh-gei#902]
In our log output in `gh bbs2gh` migrations, we say that:

> Your migration archive should be ready on your instance at $BITBUCKET_SHARED_HOME/data/migration/export/Bitbucket_export_{exportId}.tar

https://github.com/github/gh-gei/blob/a8ddc6960494f5b57a5eb86497bed477a3ebe7ac/src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs#L181

This output is a bit confusing because the environment variable that Bitbucket uses is $BITBUCKET_HOME ([source](https://confluence.atlassian.com/bitbucketserver/set-the-home-directory-776640890.html)), not $BITBUCKET_SHARED_HOME.

We should change this to be more consistent with the terms that Bitbucket itself uses.

As part of this change, we may also want to consider renaming the `--bbs-shared-home` argument to `--bbs-home`.

## Todo

- [ ] ...

## Dependencies

- ...

**Comment by ArinGhazarian** [comment]
@timrogers `$BBS_HOME` is quite different than`$BBS_SHARED_HOME`. BBS home is the location where Bitbucket data is but the shared home is the `shared` folder which is located in `{$BBS_HOME}/bitbucket/shared`. Our source of truth is [this doc](https://confluence.atlassian.com/bitbucketserver/exporting-957497835.html) that used the `$BBS_SHARED_HOME`. In the doc, under `Performing the export` you can see the following:
> The archive file can be found after the migration is finished, in $BITBUCKET_SHARED_HOME/data/migration/export/Bitbucket_export_<Job ID>.tar

**Comment by timrogers** [comment]
You're right! I misunderstood the Atlassian documentation, but I see what I missed now.

But I'm still wondering if we should change these references to refer to `BITBUCKET_HOME` instead. I agree that `BITBUCKET_SHARED_HOME` is mentioned in the Bitbucket API docs, but it is much more common to talk about `BITBUCKET_HOME` instead. There are only 4 Google results for the former, and 6,320 for the latter.

That said, I don't think this issue is super important. A good solution to #903 will be much more valuable than this change.

**Comment by ArinGhazarian** [comment]
@timrogers, I think whether we change it or not depends on the people's familiarity with Bitbucket administration. I was under the impression that the `$BITBUCKET_HOME` (or `$BITBUCKET_SHARED_HOME`) would already be an existing environment variable created during the BBS setup but that wasn't the case!  I confirmed it by checking the environment variables on one of our BBS test servers. IMO our source of truth should still be the BBS migration docs but based on feedback we can determine which one makes more sense.
Those being said, we made it clear what BBS shared home means in `--bbs-shared-home` option's help text:
> Bitbucket server's shared home directory. Defaults to "/var/atlassian/application-data/bitbucket/shared" if downloading the archive from a 
server using SSH and "c$\atlassian\applicationdata\bitbucket\shared" if downloading using SMB.
